tests/e2e/testfiles/powerscale-templates/powerscale-secret-template.yaml

```diff
@@ -5,4 +5,4 @@ isilonClusters:
     isDefault: true
     endpoint: "REPLACE_ENDPOINT"
     skipCertificateValidation: true
-    endpointPort: 8080
+    endpointPort: "REPLACE_PORT"
```

Comment:
... and here we have it inside quotation marks. Does it matter that it's in quotes sometimes and not in quotes others?

Reply:
It doesn't matter. Tested with and without, both approaches worked but it was last tested with quotes in this scenario.